### PR TITLE
Fix static analysis issues

### PR DESCRIPTION
**Pull request by kouvel**

### CODE REVIEW

**Comment by kouvel** [comment]
@janvorli @gkhanna79

File: src/md/ceefilegen/cceegen.cpp [hunk 1]
```diff
@@ -47,10 +47,20 @@ HRESULT CCeeGen::CreateNewInstance(CCeeGen* & pGen) // static, public
     TESTANDRETURNMEMORY(pGen->m_peSectionMan);
 
     HRESULT hr = pGen->m_peSectionMan->Init();
-    TESTANDRETURNHR(hr);
+    if (FAILED(hr))
+    {
+        pGen->Cleanup();
+        delete pGen;
```
Comment (janvorli) [inline src/md/ceefilegen/cceegen.cpp]: Can you please use NewHolder<CCeeGen> as a local and use its Extract method to the output parameter in the success case? It is preferred to use holders in the runtime, especially when there are multiple return code paths.

File: src/vm/ceeload.cpp [hunk 2]
```diff
@@ -12757,6 +12757,7 @@ void Module::LogTokenAccess(mdToken token, SectionFormat format, ULONG flagnum)
         GC_NOTRIGGER;
         MODE_ANY;
         PRECONDITION(g_IBCLogger.InstrEnabled());
+        PRECONDITION(flagnum >= 0);
```
Comment (janvorli) [inline src/vm/ceeload.cpp]: The flagnum is ULONG so I am not sure how can this comparison help anything. What was the issue?

File: src/zap/zapheaders.cpp [hunk 3]
```diff
@@ -325,7 +325,7 @@ ZapPEExports::ZapPEExports(LPCWSTR dllPath)
 
 DWORD ZapPEExports::GetSize()
 {
-	return DWORD(sizeof(IMAGE_EXPORT_DIRECTORY) + wcslen(m_dllFileName) + 1);
+	return DWORD(sizeof(IMAGE_EXPORT_DIRECTORY) + wcslen(m_dllFileName) * sizeof(BYTE) + 1);
```
Comment (janvorli) [inline src/zap/zapheaders.cpp]: This change looks strange. Multiplying the wcslen - length of 16 bit string -  by sizeof BYTE?

File: src/md/ceefilegen/cceegen.cpp (hunk 1 above)
Comment (kouvel) [inline src/md/ceefilegen/cceegen.cpp]: Done

File: src/vm/ceeload.cpp (hunk 2 above)
Comment (kouvel) [inline src/vm/ceeload.cpp]: You're right, I have undone the changes relevant to flagnum. The error was about tkKind.

File: src/zap/zapheaders.cpp (hunk 3 above)
Comment (kouvel) [inline src/zap/zapheaders.cpp]: It didn't like combining sizeof (byte count) and char length (element count). The scaling is to explicitly indicate that the intention is to use only one byte per char, as done in the Save() function below.

**Pull request merged**